Automattic/vip-design-system
# Add @storybook/addon-storysource to simplify code examples

## Description

This addon adds the source code of the stories into a new tab. This is great for copying the examples and use in the VIP Dashboard / etc.

## Checklist


## Steps to Test

1. Pull down PR.
2. `npm run dev`.
3. Open Storybook.
3. Check the Story tab on any component
4. You should see the source code of a story

![image](https://user-images.githubusercontent.com/3402/183471542-10c0bf98-19d2-456e-8f8b-3a4aa1ca2411.png)
